src/containers/HomePage/HomePage.js

```diff
@@ -4,10 +4,10 @@ import '../../stylesheets/styles.scss';
 
 const HomePage = () => {
 	return (
-		<div className="app-container">
+		<Fragment>
```

Comment:
can't believe what I saw

Reply:
Is it wrong?

Reply:
just impressed that you used `<fragment />`